accounts/emailer.py

```diff
@@ -0,0 +1,20 @@
+from django.core.mail import EmailMultiAlternatives
+from djangoRestReactShop.settings import EMAIL_HOST_USER, BASE_WEBSITE_URL
+
+
+def send_email(email, password):
+    message = ''
+    message += f'<div style="max-width: 320px;">'
+    message += f'<br>'
+    message += f'<a href="{BASE_WEBSITE_URL}" target="_blank" style="font-size: 15px;">{BASE_WEBSITE_URL}</a>'
+    message += f'<h3>Congratulations! You have successfully registered!</p>'
+    message += f'<br>'
+    message += f'<p>Email: { email }</p>'
+    message += f'<p>Password: { password }</p>'
+    message += f'<div/>'
```

Comment:
This should look much neater
```suggestion
    message = (
        f'<div style="max-width: 320px;">'
        "<br>"
        '<a href="{BASE_WEBSITE_URL}" target="_blank" style="font-size: 15px;">{BASE_WEBSITE_URL}</a>'
        "<h3>Congratulations! You have successfully registered!</p>"
        "<br>"
        "<p>Email: {email}</p>"
        "<p>Password: {password}</p>"
        "<div/>"
    )
```

Also please use `{email}` rather than `{ email }` in f-strings.
But in HTML templates you should use `{{ email }}` instead of `{{email}}`

Reply:
Исправил